redis/tests/support/mod.rs

```diff
@@ -45,6 +45,10 @@ enum ServerType {
     Unix,
 }
 
+pub enum RedisModule {
```

Comment:
Naming nit: I think the `Redis` in both `RedisJson` and `RedisModule` is superfluous, so let's ditch that?